experimenter/experimenter/targeting/constants.py

```diff
@@ -180,6 +180,35 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_WINDOWS_1903_EXCLUDE_RTAMO = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default w/ pin, excluding users coming from RTAMO"
+    ),
+    slug="first_run_new_profile_exclude_rtamo",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default w/ pin, excluding RTAMO"
+    ),
+    targeting=(
+        "{first_run} && {has_pin} && {attribution}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
+            has_pin=HAS_PIN,
+            attribution="attributionData.source != 'addons.mozilla.org'",
+        )
+    ),
+    desktop_telemetry=(
+        "{first_run} AND {has_pin} AND {attribution}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.desktop_telemetry,
+            has_pin=HAS_PIN,
+            attribution="attributionData.source != 'addons.mozilla.org'",
```

Comment:
I don't think this is actually used, but it's confusing that you've mixed targeting syntax with telemetry syntax. https://github.com/mozilla/experimenter/blob/d6a782f7e8b2b55c4eebb96d9b33cd80b09b2f81/experimenter/experimenter/targeting/constants.py#L81-L83